java-symbol-solver-core/src/test/java/com/github/javaparser/symbolsolver/model/typesystem/TypeVariableUsageTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Federico Tomassetti
```

Comment:
But if the file has undergone a lot of changes by different authors this line dosn't tell you anything. And you get back to looking at the commit history ;)
If we want to keep this line for legal purposes I'd suggest that we change it to "Contributors of JavaSymbolSolver" because that is aligned with the project's Apache License.